contribs/gmf/src/search/component.js

```diff
@@ -345,13 +345,6 @@ export class SearchController {
      */
     this.clearButton = this.options.clearButton !== false;
 
-    if (this.clearButton) {
```

Comment:
This must be `!this.clearButton`. It's what has changed, by my fault, since the 2.5. And it's why the search works bad on both desktop and desktop_alt interfaces.

The aim of the block in the if is to clear the input on focus and on blur (blur means "leave focus", you could add a comment about that here).
We don't want to change this behavior. I would keep this block.